packages/db/src/core/integration/load-astro-config.ts

```diff
@@ -69,8 +69,8 @@ async function loadConfigWithVite(configPath: string): Promise<Record<string, un
 
 async function createViteServer(): Promise<ViteDevServer> {
 	const viteServer = await createServer({
-		server: { middlewareMode: true, hmr: false, watch: { ignored: ['**'] } },
-		optimizeDeps: { disabled: true },
+		server: { middlewareMode: true, hmr: false, watch: null },
+		optimizeDeps: { noDiscovery: true },
```

Comment:
This change keeps it in line with the current code in Astro core, however, it doesn't look like this file is used at all. Is it safe to remove it?

Reply:
Ben mentioned on Discord that the file is no longer used as the CLI attaches to the `astro` commend. So I've removed it in the last commit.